mix.exs

```diff
@@ -1,44 +1,47 @@
 defmodule ExJsonSchema.Mixfile do
   use Mix.Project
 
+  @source_url "https://github.com/jonasschmidt/ex_json_schema"
+  @version "0.7.4"
+
   def project do
     [
       app: :ex_json_schema,
-      version: "0.7.4",
-      elixir: "~> 1.3",
-      description:
-        "A JSON Schema validator with full support for the draft 4 specification and zero dependencies.",
+      version: @version,
+      elixir: "~> 1.6",
```

Comment:
This is the only part I'm not sure about. I only removed versions `< 1.6` from the Travis build matrix because the build was taking ages otherwise. I'm thinking about re-adding one build for the oldest Elixir version that's supported (= `1.3`). I wouldn't want to lock out users running on older Elixir versions unless there's a compelling reason to do so.

Reply:
I've reverted the change back to Elixir 1.3.

Testing through `asdf` with version below:
```
$ cat .tool-versions 
erlang 18.3
elixir 1.3.4-otp-18
```

Just checking, when I ran `mix test`, it seems to raise this error?
```
$ mix test
...
Generated poison app
==> jason
warning: the dependency :jason requires Elixir "~> 1.4" but you are running on v1.3.4
...
==> ex_json_schema
Compiling 12 files (.ex)
Generated ex_json_schema app

=ERROR REPORT==== 8-Feb-2021::23:18:23 ===
Failed initiating web server: 
undefined
{invalid_option,
    {non_existing,{document_root,"./test/JSON-Schema-Test-Suite/remotes"}}}

** (MatchError) no match of right hand side value: {:error, {{:shutdown, {:failed_to_start_child, {:httpd_manager, :any, 1234, :default}, {:error, {:invalid_option, {:non_existing, {:document_root, './test/JSON-Schema-Test-Suite/remotes'}}}}}}, {:child, :undefined, {:httpd_instance_sup, :any, 1234, :default}, {:httpd_instance_sup, :start_link, [[bind_address: :any, ipfamily: :inet, server_name: 'test 1', document_root: './test/JSON-Schema-Test-Suite/remotes', server_root: '.', port: 1234], 15000, []]}, :permanent, :infinity, :supervisor, [:httpd_instance_sup]}}}
    test/test_helper.exs:4: (file)
    (elixir) lib/code.ex:363: Code.require_file/2
    (elixir) lib/enum.ex:651: Enum."-each/2-lists^foreach/1-0-"/2
    (elixir) lib/enum.ex:651: Enum.each/2
    (mix) lib/mix/tasks/test.ex:228: Mix.Tasks.Test.run/1
```

Reply:
Ok, so that seems to be an issue with the test setup that starts the `:inets` webserver that serves remote schemata. Looks like it can't find the document root, so it might be possible to solve this by resolving and giving it an absolute path 🤔

On the other hand, if `jason` only emits a warning when compiling with an Elixir `<` than what its `mix.exs` specifies, I'm also fine with leaving the version constraint as `~> 1.6` in `mix.exs` here. I was under the impression that the constraint would prevent compilation of the library with an older Elixir version 👍

Reply:
> I was under the impression that the constraint would prevent compilation of the library with an older Elixir version +1

Noted. Set to `-> 1.6` then.